microdnf/commands/advisory/advisory_subcommand.cpp

```diff
@@ -60,6 +60,16 @@ AdvisorySubCommand::AdvisorySubCommand(
     updates->arg->set_conflict_arguments(conflict_args);
 }
 
+void AdvisorySubCommand::add_running_kernel_packages(libdnf::Base & base, libdnf::rpm::PackageQuery & package_query) {
```

Comment:
It's not entirely clear to me why this is needed, maybe a short comment with an explanation for this would be helpful?

Reply:
Added a comment there. :+1: